app/start.js

```diff
@@ -5,6 +5,8 @@ import { theCivilServer, Iota } from 'civil-server'
 import civilIotas from '../node_modules/civil-server/iotas.json'
 import iotas from '../iotas.json'
 import App from './components/app'
+import getSitemap from './tools/getSitemap'
```

Comment:
Changes to this file shouldn't be required.   the files in app/routes will automatically get loaded. 
The sitemap should get generated by app/routes/sitemap